.github/workflows/build-release.yml

```diff
@@ -0,0 +1,75 @@
+name: Release
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
Would this run on a tag on any branch? I'm thinking we'd want to only run for tags on the master branch.